cmake/Hunter/config.cmake

```diff
@@ -30,7 +30,7 @@ hunter_config(
 )
 
 hunter_config(libp2p
-    URL https://github.com/soramitsu/libp2p/archive/00ec26288abe3c70c3fab4b0529e02302a77fd20.zip
-    SHA1 b818125aa885a877d1f51836b8f632689b25766b
+    URL https://github.com/soramitsu/libp2p/archive/1e874e7eefb45b0b47e00aa73334411b031e385b.zip
```

Comment:
Just a heads-up: this will conflict with https://github.com/filecoin-project/cpp-filecoin/pull/26/files (depending which one gets merged first)